learningequality/studio
# Channel main tree fix

## Description

The primary purpose of this PR is to rectify the non-creation of the main tree object upon channel creation.

This PR adds this to the create serializer, and also propagates the results of this back to the frontend - as such, this provides a demonstration of how to use the `sync` endpoint changes mechanism.

In addition, this PR:
* Removes sort_order assignment on node creation, as this attribute is no longer used for anything.
* Fixes the reference check for `Raven` in `client.js` - this was failing because it was referencing an undefined variable, not an undefined property of an appropriate reference.

## Steps to Test

- Create a channel
- Confirm that after syncing, the channel object in indexeddb has a `root_id` property
